examples/quantization_aware_training/torch/resnet18/main.py

```diff
@@ -0,0 +1,325 @@
+        train_dir,
+        transforms.Compose(
+            [
+                transforms.Resize(IMAGE_SIZE),
+                transforms.RandomHorizontalFlip(),
+                transforms.ToTensor(),
+                normalize,
+            ]
+        ),
+    )
+    val_dataset = datasets.ImageFolder(
+        val_dir,
+        transforms.Compose(
+            [
+                transforms.Resize(IMAGE_SIZE),
+                transforms.ToTensor(),
+                normalize,
+            ]
+        ),
+    )
+
+    train_loader = torch.utils.data.DataLoader(
+        train_dataset, batch_size=BATCH_SIZE, shuffle=True, num_workers=0, pin_memory=True, sampler=None
+    )
+
+    val_loader = torch.utils.data.DataLoader(
+        val_dataset, batch_size=BATCH_SIZE, shuffle=False, num_workers=0, pin_memory=True
+    )
+
+    # Creating separate dataloader with batch size = 1
+    # as dataloaders with batches > 1 are not supported yet.
+    calibration_dataset = torch.utils.data.DataLoader(
+        val_dataset, batch_size=1, shuffle=False, num_workers=0, pin_memory=True
+    )
+
+    return train_loader, val_loader, calibration_dataset
+
+
+def prepare_tiny_imagenet_200(dataset_dir: Path):
+    # Format validation set the same way as train set is formatted.
+    val_data_dir = dataset_dir / "val"
+    val_images_dir = val_data_dir / "images"
+    if not val_images_dir.exists():
+        return
+
+    val_annotations_file = val_data_dir / "val_annotations.txt"
+    with open(val_annotations_file, "r") as f:
+        val_annotation_data = map(lambda line: line.split("\t")[:2], f.readlines())
+    for image_filename, image_label in val_annotation_data:
+        from_image_filepath = val_images_dir / image_filename
+        to_image_dir = val_data_dir / image_label
+        if not to_image_dir.exists():
+            to_image_dir.mkdir()
+        to_image_filepath = to_image_dir / image_filename
+        from_image_filepath.rename(to_image_filepath)
+    val_annotations_file.unlink()
+    val_images_dir.rmdir()
+
+
+def run_benchmark(model_path: str, shape: Optional[List[int]] = None, verbose: bool = True) -> float:
+    command = f"benchmark_app -m {model_path} -d CPU -api async -t 15"
+    if shape is not None:
+        command += f' -shape "[{",".join(str(x) for x in shape)}]"'
+    cmd_output = subprocess.check_output(command, shell=True)  # nosec
+    if verbose:
+        print(*str(cmd_output).split("\\n")[-9:-1], sep="\n")
+    match = re.search(r"Throughput\: (.+?) FPS", str(cmd_output))
+    return float(match.group(1))
+
+
+def get_model_size(ir_path: str, m_type: str = "Mb", verbose: bool = True) -> float:
+    xml_size = os.path.getsize(ir_path)
+    bin_size = os.path.getsize(os.path.splitext(ir_path)[0] + ".bin")
+    for t in ["bytes", "Kb", "Mb"]:
+        if m_type == t:
+            break
+        xml_size /= 1024
+        bin_size /= 1024
+    model_size = xml_size + bin_size
+    if verbose:
```

Comment:
1) We need to update all samples because I just copied this function from other torch example. To be consistent we need to do this in a separate PR.
2) I'm not sure we can post any numbers in the README as it is against our company policy

Reply:
1. Ofc, not in this PR.
2. Openivno provided [performance](https://docs.openvino.ai/2023.3/openvino_docs_performance_benchmarks.html#), i think with disclaimer we can add it

@MaximProshin  @alexsu52  What you think about format of example`s output and summary with performance in readme?

PS. Updating another examples good issue for https://github.com/openvinotoolkit/openvino/issues

Reply:
You can implement this format for this example, but i don't see value from changes in the other samples, because each sample is standalone application.

Reply:
Now output of the sample looks perfect IMHO, done